### ISSUE TITLE
(URGENT) In annotated EXPRESS schemas, general, fund_cons and express-g blocks should all be separately tagged

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/stepmod-utils#69]
We need to adopt the new structure described in metanorma/annotated-express#38. This is urgent because of the workflow we have now where @opoudjis imports content from iso-10303-stepmod into this repository.

See metanorma/annotated-express@acfee797e525465532cc10008503314c925f5b11 for the changes needed.

1.
```diff
 (*"action_schema"
-== General
 
-The subject of the *action_schema* is the description of actions, the reasons for these actions, and the status of these actions. 
+The subject of the *action_schema* is the description of actions, the reasons for these actions, and the status of these actions.
```

2.
```diff
-== Fundamental concepts and assumptions
+*)
+
+(*"action_schema.__fund_cons"
 
 Action information can be attached to any aspect of product data.
```

3.
```diff
+(*"action_schema.__expressg"
 [.svgmap]
 ====
-[[action_schemaexpg1]]
 image::action_schemaexpg1.svg[]
```

```diff
+*)
+
+(*"action_schema.__expressg"
 [.svgmap]
 ====
-[[action_schemaexpg2]]
 image::action_schemaexpg2.svg[]
```

**Comment by ronaldtse** [comment]
This is the specification: https://github.com/metanorma/annotated-express/issues/38#issuecomment-803263905

**Comment by ronaldtse** [comment]
Fixed in #70.
